lib/registrant.rb

```diff
@@ -0,0 +1,57 @@
+require 'pry'
+
```

Comment:
What happened to the indentations in this file? The other files had spacing set to 2, this one has it set to 4. 
This can happen when cloning down a repo. Should be a setting in VS Code to make it consistent. We want to maintain that consistency throughout the whole project. I prefer a 2-count tab, but that's just me.